src/day-1-toy/fileio.py

```diff
@@ -1,12 +1,19 @@
 # Use open to open file "foo.txt" for reading
+f = open('foo.txt', 'r')
 
 # Print all the lines in the file
+print(f.read())
 
 # Close the file
-
+f.close()
 
 # Use open to open file "bar.txt" for writing
+o = open('bar.txt', 'w')
 
 # Use the write() method to write three lines to the file
```

Comment:
you can do this with a loop
`with open("bar.txt", "w") as f:
  for line in lines:
    f.write(line)
    f.write("\n")`